css-two-column-layout/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body class="display-and-center">
+  <div class="container">
+    <div class="row row-one">
+      <h1>Scale, Speed, Agility</h1>
+      <p class="row-one-text">We've dedicated thousands of man hours to making health care institutions in the Helium network run
+        10x more efficiently and deliver an unparalleled quality of care.
+      </p>
+    </div>
+    <div class="row row-two">
+      <div class="half-column column-one">
+        <div class="row-in-column">
```

Comment:
You seem to have multiple different variations of the `row` class, such as `row-one`, `row-two`, and `row-in-column`. You should just reuse the `row` class throughout your document, and make additional classes for any additional styling you need to apply.